src/blr/cli/buildmolecules.py

```diff
@@ -17,17 +17,18 @@
 
 def main(args):
     summary = Summary()
+    out_mode = utils.get_output_mode(args.output, args.output_format)
```

Comment:
If we have an option for specifying output format I think that should override the filename.

```suggestion
    if not args.output_format:
        out_mode = utils.get_output_mode(args.output, args.output_format)
```